### DISCUSSION TITLE
User Types like Entry Types

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [craftcms/cms#13442] [Ideas] [answered]
I had raised the point before. Nevertheless, I am raising the issue again.

Users should also have entry types (user types) so that different data sets can be entered depending on the user.

We use Craft CMS partly as a kind of CMS to manage our customers, suppliers but also our team members and freelancers. Depending on the type, we have different requirements and information that should be attached to a user. 

Unfortunately, it is currently a bit confusing without specific types.

Creating everything with conditions is also a bit tedious, since Craft already has the principle, it would be very practical to have this for the users as well. Two customers have already asked us about this.

**Comment by brandonkelly** [comment, marked as answer]
It seems like it would be pretty confusing to have both User Groups _and_ User Types.

> Creating everything with conditions is also a bit tedious

Craft 5 will add support for reusing custom fields multiple times within a single field layout. Which means you could start creating user “type”-specific tabs that contain a tab-level condition (e.g. based on a custom Dropdown field, or the selected user group(s)), and pull in whatever custom fields you need to those tabs, even if they were already pulled into another type’s tab. I’m guessing that will make things a bit less tedious?

**Reply by alexanderbuergin** [reply]
Yes, I think that is a good way to move forward.

As you say, classic Types would be a bit too much, when we already have user groups. That I would propose is to plan this in a way that Tabs can be shown to different user groups.

That would be nice as well, is the option to move the default files that every user have in the tag/field layout
